src/Infrastructure.VS.UnitTests/ThreadHandlingTests.cs

```diff
@@ -22,20 +22,13 @@
 using System.Threading.Tasks;
 using FluentAssertions;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
-using SonarLint.VisualStudio.Core;
 using SonarLint.VisualStudio.Integration.UnitTests;
 
 namespace SonarLint.VisualStudio.Infrastructure.VS.UnitTests
 {
     [TestClass]
     public class ThreadHandlingTests
     {
-        [TestMethod]
-        public void MefCtor_CheckIsExported()
-        {
-            MefTestHelpers.CheckTypeCanBeImported<ThreadHandling, IThreadHandling>(null, null);
-        }
-
```

Comment:
Cleanup. We don't actually want or need to MEF import this type.

Reply:
We do, in CFamily code, i.e. VcxRequestFactory -- this is part of the fix I've implemented earlier.

Reply:
Ah, I missed that when I search for usages. I'll change the `VcxRequestFactory`.